_homework/interaction-design/sketch-symbols.md

```diff
@@ -4,17 +4,43 @@ assigned: Aug 31, 2018
 due: Sept 4, 2018
 ---
 
-**Note:** More details coming soon.
+Now that you have built some cards and changed photos, descriptions and titles using nested symbols, we'd like for you to see how else you can implement them in your designs! 
 
+**Upload your final Sketch files** [to the Google Drive](https://drive.google.com/drive/u/1/folders/1OJWSXlXKHCHchXesu9GJhFflCIS-RvV0) in the "Completed Work" folder.
 
-Sketch Tutorials
-----------------
 
+1st Deliverable: Design a Button System
+-------------------------------------------------
 
-Exercise
---------
+1. Watch [Creating a Button System with Nested Symbols on Sketch App](https://www.youtube.com/watch?v=_bjqVF7Fvg4&t=794s) from Sketch Together <small>Duration: 30:39</small>
+2. Create your own button system using nested symbols, masks, and overrides. 
+3. Get creative! Use colors, icons and text of your choice. 
 
+### Requirements
+
+- Primary and secondary buttons
+- At least 3 different states (i.e. active, disabled, hover, pressed, in progress, completed, error, etc.)
+- Icon in at least 1 state.
+
+### Nice to Have
+
+- At least 2 different sizes (i.e. large, medium, or small)
+
+
+2nd Deliverable: Design a Bottom Nav Bar
+-------------------------------------------------
+
+For this exercise, you will create a bottom nav bar for a mobile app similar to one of these:
```

Comment:
I'm guessing there's meant to be a list after this :)

Reply:
Yes, I want to add some photos but don't know how to do that. Help, please! :)